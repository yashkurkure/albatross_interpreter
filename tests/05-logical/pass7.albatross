if (!(103*157&228|87-42-53*97|79+174|123&109-10+68-211*166|2000000)>103*157&228|87-42-53*97|79+174|123&109-10+68-211*166) {if (!228-32*251&98&6|255|46+119*112+115&17*136&16==228-32*251&98&6|255|46+119*112+115&17*136&16) {if (!155-51|214|142|141|94|97+55|216&204&233-88|79|251-87==155-51|214|142|141|94|97+55|216&204&233-88|79|251-87) {if (!198*162|120|108*104-120*190&87+198|222&110*66|48&179<(198*162|120|108*104-120*190&87+198|222&110*66|48&179|2000000)) {if ((159*122*251|158-203+84+16|37|140|2000000)>159*122*251|158-203+84+16|37|140) {printint( 9*170+50+206|219|129&6);}}}}}
if (!(170|2000000)>170) {if ((128|2000000)>128) {if (128==128) {if (!225+49+88|40|205+221|140+255-68|234-98+119|16-201|74==225+49+88|40|205+221|140+255-68|234-98+119|16-201|74) {if (63|218|7*126*122|13|118|160+120*1|98-76-0&48|124|222<>122-159+248&111&94) {printint( 206|178+244*124-36&76&241&233*17*91|148&148|53-71|41*199|209|48+103*228|149&169|243|195|115|133&132-228);}}}}}
if ((170*248|13*122&222|175*72|166|118|100|187|51+21&21&90|2000000)>170*248|13*122&222|175*72|166|118|100|187|51+21&21&90) {if (!174|138+212&153-68|232|125-103+101<>66&51*24|49|94*52&200+188|5*218|223|40) {if (!160*57|214+132*182+49&124*35-15-6|144<(160*57|214+132*182+49&124*35-15-6|144|2000000)) {if (205==205) {if (!12*90*153+248+22+65+210*164-150|151*202|125+137|203*14+217<>122|21-14|5&255|90+127+61+123) {printint( 170);}}}}}
if (207|65&135|240*145|236|124-130*237-50|0+14<>220&80|1&136+127*149+254+9&115) {if (57+220|131|72*142&159|71-169*44*173|247|54|184==57+220|131|72*142&159|71-169*44*173|247|54|184) {if (!81-140|131|217*112|181+49&136|0&216*110|63|33|182*183==81-140|131|217*112|181+49&136|0&216*110|63|33|182*183) {if (!242+106-39+17+154|220|222|246&195<(242+106-39+17+154|220|222|246&195|2000000)) {if (!86-0-174&134+116+33|124+49|53|129|103&166-204-26|251<>102+111+237|170|127-223|111|225+172&13-68|110-202) {printint( 134*135|39|115*128*213*216+121*234-162*0+47|75);}}}}}
if (123|125|44|9&67+10-247|12+44==123|125|44|9&67+10-247|12+44) {if (!102&174+107*241+92|253|140|52*129-163||1) {if (7|242-44+91+115-79|74|94-1*186*19+167-81&207|105*154||1) {if (!31==31) {if (199&225&74|90-51&155+133|118+159*231|144==199&225&74|90-51&155+133|118+159*231|144) {printint( 206*153*241*97|229|207|74*152-138&192+1-155*228*175-224-147|63&51|6+75+226|21*10+0&149);}}}}}
if (!164&162*228*238*199+242|217|103+141-51+224&86-39&194==164&162*228*238*199+242|217|103+141-51+224&86-39&194) {if (!1*114|207|167&66&19-21*210*147+164-183&93|112==1*114|207|167&66&19-21*210*147+164-183&93|112) {if (143|100|128*69+60-154|119-15|96|43&126-248|127-26&109+87<(143|100|128*69+60-154|119-15|96|43&126-248|127-26&109+87|2000000)) {if (!29<(29|2000000)) {if (143&100+78+99&242*9|118-235-230-250*234-174-70-173+110||1) {printint( 172&122|10&135|82&121&110|72|102*243|17|173*174|248-146|186+207*213-200*44);}}}}}
if (!186*201&145&158|107*193==186*201&145&158|107*193) {if (!132*61+86+119|63|144*223-82*205|42&116<(132*61+86+119|63|144*223-82*205|42&116|2000000)) {if (35|89|227&17+143|41-147-96&188+46==35|89|227&17+143|41-147-96&188+46) {if (!140-67&149*248*192-104<>118) {if (!(189-247-68|81-248|193|190-16&241+78|206+25|53|2000000)>189-247-68|81-248|193|190-16&241+78|206+25|53) {printint( 108-79-210|223*46|236-121-21*87+152+98|24+61-146|205+89);}}}}}
if (!144*228|66&11|23+101&1-63+42<(144*228|66&11|23+101&1-63+42|2000000)) {if (!12*16+171*113*23|190|86|74*85*212==12*16+171*113*23|190|86|74*85*212) {if (153|39|3*241|62-209|57|123&188&178|132-225|200+57|19+118==153|39|3*241|62-209|57|123&188&178|132-225|200+57|19+118) {if (!109|131-90|16+109&55|120&215*52&200|242*80|132|6-119+197<(109|131-90|16+109&55|120&215*52&200|242*80|132|6-119+197|2000000)) {if (169*121+193|75|39+35+128|131|106|58|87&247&193==169*121+193|75|39+35+128|131|106|58|87&247&193) {printint( 79+95|239|173&91&51&235-207|190*151+143*142-153+145+57);}}}}}
if ((91|163*28-147*193*224&160|100+47*197*133&197|2000000)>91|163*28-147*193*224&160|100+47*197*133&197) {if (!137-121|196|30*228|238&223|217+254+190-191&159+15<(137-121|196|30*228|238&223|217+254+190-191&159+15|2000000)) {if (95*248*197|194-225&108&4&50|135|8|64+121&126==95*248*197|194-225&108&4&50|135|8|64+121&126) {if (203&4|170|8&13-150|128*150==203&4|170|8&13-150|128*150) {if (251-85|30-2-24|229&164+226==251-85|30-2-24|229&164+226) {printint( 104+228|194-61|236|124*21+152|120|85+228-22|25+158&148+164-249-65|216|251|34|223+224*188*118-156-102*4*93&60);}}}}}
if (64+122*67*155*28-5*164+6|183==64+122*67*155*28-5*164+6|183) {if (!(135|108&32*80-204|199|8|218&32*123-206|2000000)>135|108&32*80-204|199|8|218&32*123-206) {if (!236|254-196|247|45*6-203|242&232*11*176|207-104&106|188|216<>67|17+166&148+238|111*138+214+11) {if (189&6+212&203&20|177|142&103|239==189&6+212&203&20|177|142&103|239) {if (252-102*59|73|10&195-99+58&37|153|246|214|183<(252-102*59|73|10&195-99+58&37|153|246|214|183|2000000)) {printint( 98+214+12&171-99-7|233-224&195|44&207+3+70-232*54&86&104*204*16|139|4|248*73);}}}}}
exit(0);
