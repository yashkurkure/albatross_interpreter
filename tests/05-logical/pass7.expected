178717067726718365535-149151307118175
